src/test/java/com/epam/atm/selenium/BaseActionTest.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.atm.selenium;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+public class BaseActionTest {
```

Comment:
We need to change this extension of classes

Have one **abstract** BaseTest
move all common in it
    protected ChromeDriver driver;
    protected String LOGIN = "yauhen_valodzin@epam.com.qa";
    protected String PASSWORD = "Ceakt_1234";

**protected abstarct getWebDriver();**

    @BeforeClass(description = "Start browser")
    public void startBrowser() {

Here this part is different, we will move it to another method -> protected abstarct getWebDriver();
        **System.setProperty("webdriver.chrome.driver", "C:\\data\\tools\\chromedriver.exe");**
        **driver = new ChromeDriver();**
        
        driver.manage().window().maximize();
        System.out.println("Maximize window");
        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
        System.out.println("Add Implicit wait");
    }

    @AfterClass(description = "Close browser")
    public void closeBrowser() {
        driver.quit();
        System.out.println("Browser was closed");
    }


Create to child classes like BaseLocalTest and BaseRemoteTest
add getWebDriver() implementation based on what u want from driver